apps/server/src/routers/api/mod.rs

```diff
@@ -17,12 +18,40 @@ mod tests {
 		NamedType,
 	};
 
-	use crate::config::jwt::CreatedToken;
-
-	use super::v1::{
-		auth::*, book_club::*, emailer::*, epub::*, job::*, library::*, media::*,
-		metadata::*, series::*, smart_list::*, user::*, ClaimResponse, StumpVersion,
-		UpdateCheck,
+	use crate::{
+		config::jwt::CreatedToken,
+		routers::api::v1::{
+			auth::{AuthenticationOptions, LoginOrRegisterArgs, LoginResponse},
+			book_club::{
+				BookClubInvitationAnswer, CreateBookClub, CreateBookClubInvitation,
+				CreateBookClubMember, CreateBookClubSchedule, CreateBookClubScheduleBook,
+				CreateBookClubScheduleBookOption, GetBookClubsParams, UpdateBookClub,
+				UpdateBookClubMember,
+			},
+			emailer::{
+				CreateOrUpdateEmailDevice, CreateOrUpdateEmailer, EmailerIncludeParams,
+				EmailerSendRecordIncludeParams, PatchEmailDevice,
+				SendAttachmentEmailResponse, SendAttachmentEmailsPayload,
+			},
+			epub::{CreateOrUpdateBookmark, DeleteBookmark},
+			job::UpdateSchedulerConfig,
+			library::{
+				CleanLibraryResponse, CreateLibrary, LibraryStatsParams,
+				PatchLibraryThumbnail, UpdateLibrary, UpdateLibraryExcludedUsers,
+			},
+			media::{
+				individual::{MediaIsComplete, PutMediaCompletionStatus},
+				thumbnails::PatchMediaThumbnail,
+			},
+			metadata::MediaMetadataOverview,
+			series::{PatchSeriesThumbnail, SeriesIsComplete},
+			smart_list::{
+				CreateOrUpdateSmartList, CreateOrUpdateSmartListView,
+				GetSmartListsParams, SmartListMeta, SmartListRelationOptions,
+			},
+			user::{CreateUser, DeleteUser, UpdateUser, UpdateUserPreferences},
+			ClaimResponse, StumpVersion, UpdateCheck,
+		},
```

Comment:
I wasn't really sure if I should stick with wildcard imports here - I could switch it to use them. 

Typically I don't use wildcards unless the intended way to read it is "everything in this module belongs in this scope" That wasn't the case in this instance since most of the module components are Axum handlers and don't get used here.

It's definitely a lot more verbose this way, so I can change it back if that's preferred.